### PR TITLE
Amls 4838 - Bug fix: Validation error messages not linked to group on MSB 'Where do you send the largest amounts of money' page

### PR DESCRIPTION
**Pull request by slavvco**
In this pull request there are implemented changes to SendTheLargestAmountsOfMoney model in order to display validation messages in similar way like it was done for MostTransactions. Updates to model, view, controller, specs and conversion methods. Loaded applications saved before changes and submitted them without problem, looks like there are no issues with compatibility.

## Related / Dependant PRs?

- PR in acceptance tests for that

## How Has This Been Tested?

- unit tests
- manual checks
- regression

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [ ] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [x] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/models/moneyservicebusiness/SendTheLargestAmountsOfMoney.scala [hunk 1]
```diff
@@ -16,46 +16,89 @@
...
-      )
+  val formW: Write[SendTheLargestAmountsOfMoney, UrlFormEncoded] = {
+    import jto.validation.forms.Writes._
+    import utils.MappingUtils.spm
+    implicitly
+  }
+
+  val jsonW = Writes[SendTheLargestAmountsOfMoney] { lom =>
+    lom.countries match {
+      case Seq(a, b, c) => Json.obj("country_1" -> a, "country_2" -> b, "country_3" -> c)
```
Comment (drewmiley) [inline app/models/moneyservicebusiness/SendTheLargestAmountsOfMoney.scala]: I think this is the best way as it is clearer about its intention

File: app/services/LandingService.scala [hunk 2]
```diff
@@ -283,7 +283,11 @@ class LandingService @Inject() (
             (_.transactionsInNext12Months.map(t => TransactionsInLast12Months(t.txnAmount))),
 
           sendTheLargestAmountsOfMoney = viewResponse.msbSection.fold[Option[SendTheLargestAmountsOfMoney]](None)
-            (_.sendTheLargestAmountsOfMoney.map(s => SendTheLargestAmountsOfMoney(s.country_1, s.country_2, s.country_3))),
+            (_.sendTheLargestAmountsOfMoney.map (s => s.countries match {
+              case Seq(a, b, c) => SendTheLargestAmountsOfMoney(a, Some(b), Some(c))
```
Comment (drewmiley) [inline app/services/LandingService.scala]: Could maybe do something with .head and .tail

Comment (drewmiley) [reply app/services/LandingService.scala]: But i'm not massively keen

File: app/views/msb/send_largest_amounts_of_money.scala.html [hunk 3]
```diff
@@ -38,18 +38,24 @@
 
         @fieldset(
             legend = "msb.send.the.largest.amounts.of.money.title",
-            f = f("countries"),
+            f = f("largestAmountsOfMoney-fieldset"),
             legendHidden = true,
-            panel = false
+            panel = false,
+            classes = Seq("inline")
```
Comment (drewmiley) [inline app/views/msb/send_largest_amounts_of_money.scala.html]: Do we need this?

File: app/services/LandingService.scala (hunk 2 above)
Comment (slavvco) [reply app/services/LandingService.scala]: if you don't mind I'll leave this as there are only max three elements in seq

File: app/views/msb/send_largest_amounts_of_money.scala.html (hunk 3 above)
Comment (slavvco) [reply app/views/msb/send_largest_amounts_of_money.scala.html]: nope we don't :) I'll remove

File: app/services/LandingService.scala (hunk 2 above)
Comment (drewmiley) [reply app/services/LandingService.scala]: Agreed

**Pull request merged**